docs/docusaurus.config.js

```diff
@@ -65,14 +65,14 @@ module.exports = {
       items: [
         {
           type: "doc",
-          label: "Guides",
+          label: "Documentation",
           docId: "guides/overview",
           position: "left",
-        },
+        }, 
         {
           type: "doc",
-          label: "Examples",
-          docId: "examples/introduction",
+          label: "Get Started",
+          docId: "guides/start/quickstart",
```

Comment:
This is what I meant confuses me (that two nav links point to two different anchors in the same root page.

But i know now why I was triggered, by the fact that the "Quick Start Guide" is a single item under the "Get Started" Section. 😁 

I think a quick fix would be to move the content to the menu item "Get started"

Reply:
Yes combining the sections makes sense and looks good. Having the get started on top as CTA was an idea we wanted to test.